### PR TITLE
drop support for ruby 1.x

### LINKED ISSUE #85: Running brew bundle - failure
Hello all,

On a completely fresh OSX Mountain Lion install. Homebrew is installed, brew tap homebrew/bundle completed correctly. Running: brew bundle results in the following:

```
Error: /usr/local/Library/Taps/homebrew/homebrew-bundle/lib/bundle/brew_dumper.rb:72: odd number list for Hash
        name: f["name"],
             ^
/usr/local/Library/Taps/homebrew/homebrew-bundle/lib/bundle/brew_dumper.rb:72: syntax error, unexpected ':', expecting '}'
        name: f["name"],
             ^
/usr/local/Library/Taps/homebrew/homebrew-bundle/lib/bundle/brew_dumper.rb:73: syntax error, unexpected ':', expecting '='
        full_name: f["full_name"],
                  ^
/usr/local/Library/Taps/homebrew/homebrew-bundle/lib/bundle/brew_dumper.rb:74: syntax error, unexpected ':', expecting '='
        args: args,
             ^
/usr/local/Library/Taps/homebrew/homebrew-bundle/lib/bundle/brew_dumper.rb:75: syntax error, unexpected ':', expecting '='
        version: keg["version"],
                ^
/usr/local/Library/Taps/homebrew/homebrew-bundle/lib/bundle/brew_dumper.rb:76: syntax error, unexpected ':', expecting '='
        dependencies: f["dependencies"],
                     ^
/usr/local/Library/Taps/homebrew/homebrew-bundle/lib/bundle/brew_dumper.rb:77: syntax error, unexpected ':', expecting '='
        requirements: f["requirements"],
                     ^
/usr/local/Library/Taps/homebrew/homebrew-bundle/lib/bundle/brew_dumper.rb:78: syntax error, unexpected '}', expecting '='
/usr/local/Library/Taps/homebrew/homebrew-bundle/lib/bundle/brew_dumper.rb:81: class definition in method body
/usr/local/Library/Taps/homebrew/homebrew-bundle/lib/bundle/brew_dumper.rb:89: syntax error, unexpected $end, expecting kEND
Please report this bug:
    https://git.io/brew-troubleshooting
/usr/local/Library/Taps/homebrew/homebrew-bundle/lib/bundle.rb:8:in `require'
/usr/local/Library/Taps/homebrew/homebrew-bundle/lib/bundle.rb:8
/usr/local/Library/Taps/homebrew/homebrew-bundle/cmd/brew-bundle.rb:16:in `require'
/usr/local/Library/Taps/homebrew/homebrew-bundle/cmd/brew-bundle.rb:16
/usr/local/Library/brew.rb:67:in `require'
/usr/local/Library/brew.rb:67:in `require?'
/usr/local/Library/brew.rb:155
```

### PR DESCRIPTION
**Pull request by xu-cheng**
Closes #85.

### CODE REVIEW

File: cmd/brew-bundle [hunk 1]
```diff
@@ -1,5 +1,17 @@
-#!/usr/bin/env ruby
+#!/usr/bin/env ruby -W0
 
+# Ruby version check
+unless RUBY_VERSION.split(".").first.to_i >= 2
+  abort "Ruby 2.0 or above is required. You can install it by `brew install ruby`."
```
Comment (mikemcquaid) [inline cmd/brew-bundle]: `You can install it by running` or `You can install it with`

File: cmd/brew-bundle [hunk 2]
```diff
@@ -1,5 +1,17 @@
-#!/usr/bin/env ruby
```
Comment (mikemcquaid) [inline cmd/brew-bundle]: Why rename this file? I don't mind, just curious.

Comment (xu-cheng) [inline cmd/brew-bundle]: `brew-*.rb` will be run by Homebrew's ruby interpreter, i.e. system ruby. While `brew-*` will be run as normal script. In this case, it's the user's ruby interpreter.

Comment (mikemcquaid) [inline cmd/brew-bundle]: Ah, cool :+1:

**Pull request merged**